boot/bootutil/src/ram_load.c

```diff
@@ -49,6 +49,21 @@ BOOT_LOG_MODULE_DECLARE(mcuboot);
 #endif
 #endif
 
+#ifdef MULTIPLE_EXECUTABLE_RAM_REGIONS
+int boot_get_image_exec_ram_info(uint32_t image_id,
+                                 uint32_t *exec_ram_start,
+                                 uint32_t *exec_ram_size)
+{
+
+#ifdef CONFIG_SOC_SERIES_STM32N6X
+    *exec_ram_start = DT_PROP_BY_IDX(DT_NODELABEL(axisram1), reg, 0);
+    *exec_ram_size = DT_PROP_BY_IDX(DT_NODELABEL(axisram1), reg, 1);
+#endif
+
+    return 0;
+}
+#endif /* MULTIPLE_EXECUTABLE_RAM_REGIONS */
```

Comment:
no zephyr specific stuff in common files

Reply:
yep that's fine

Reply:
Done